test/hw1/test_task4.py

```diff
@@ -0,0 +1,43 @@
+from All_home_works.hw1.task4 import find_maximal_subarray_sum
+
+
+def test_find_maximal_subarray_sum__init_data__end_subarray():
```

Comment:
Actually in that case you can remove `find_maximal_subarray_sum` from the test name because you have this information in the file.